webapp/src/ts/modules/tasks/tasks.component.ts

```diff
@@ -105,15 +105,13 @@ export class TasksComponent implements OnInit, OnDestroy {
     this.debouncedReload = _debounce(this.refreshTasks.bind(this), 1000, { maxWait: 10 * 1000 });
 
     this.currentLevel = this.sessionService.isOnlineOnly() ? Promise.resolve() : this.getCurrentLineageLevel();
-
     this.refreshTasks();
   }
 
   ngOnDestroy() {
     this.subscription.unsubscribe();
 
     this.tasksActions.setTasksList([]);
-    this.tasksActions.setTasksLoaded(false);
```

Comment:
@Benmuiruri, based on yesterday's conversation, we rolled back this change. And just keep the one from L163 to L70